src/components/TextEditor.vue

```diff
@@ -36,6 +36,7 @@ import Editor from '@ckeditor/ckeditor5-editor-balloon/src/ballooneditor'
 import EssentialsPlugin from '@ckeditor/ckeditor5-essentials/src/essentials'
 import BlockQuotePlugin from '@ckeditor/ckeditor5-block-quote/src/blockquote'
 import BoldPlugin from '@ckeditor/ckeditor5-basic-styles/src/bold'
+import ParagraphPlugin from '@ckeditor/ckeditor5-paragraph/src/paragraph'
```

Comment:
this already exist, see line 44

Reply:
good, then v1.10 doesn't need the fix, only master

my bad for opening the backport. you can just close it :rocket: